<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SouthButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EastButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QPushButton" name="West">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NorthButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TakeButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Item</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ItemDropdown">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ObtainedItems">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UseButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="ItemsInRoom_Label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Items In Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="ObtainedItems_Label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtained Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GoButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InvestigateButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Investigate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
